examples/official-storybook/stories/core.stories.js

```diff
@@ -16,13 +16,20 @@ storiesOf('Core|Parameters', module)
   .add(
     'passed to story',
     ({ parameters: { fileName, ...parameters } }) => (
-      <div>Parameters are {JSON.stringify(parameters)}</div>
+      <div>
+        <p>Parameters are</p>
+        <pre>{JSON.stringify(parameters, null, 2)}</pre>
+      </div>
     ),
     {
       storyParameter,
     }
   );
 
+storiesOf('Core|Decorator arguments', module)
```

Comment:
This is actually a new feature!

Decorators can now call the `fn` with arguments, and those arguments are collected into 1 array, and injected into the context object.

Useful? yes | no | maybe so ?

Reply:
This is interesting @ndelangen -- I'd like to hear a bit more about it.

What does `context` look like before and after in this case? What is the use case?

I suspect people will want to at least be able to change all the `parameters` to a story from a decorator. Whether or not they should be able to do that (and if it'll do what they want depending on decorator order) is questionable.